api/config.js

```diff
@@ -37,8 +39,14 @@ config.superuser.email = process.env.HACKILLINOIS_SUPERUSER_EMAIL;
 config.superuser.password = process.env.HACKILLINOIS_SUPERUSER_PASSWORD;
 
 config.auth.secret = config.secret;
-config.auth.header = 'Authorization';
+config.auth.headers = {
+  api: 'API-Authorization',
```

Comment:
What about `Authorization: "Basic [...]"` and `Authorization: "Bearer [...]"`? You could add another key for auth types in the `config.auth` dictionary, and just set `config.auth.headers.all` to be `Authorization` so that it's consistent.

I'm not sure if generalizing it with `Bearer` will mess things up for you, but that's how I've usually seen it done.